Formula/libbitcoin-node.rb

```diff
@@ -23,8 +24,7 @@ class LibbitcoinNode < Formula
 
   def install
     ENV.prepend_path "PKG_CONFIG_PATH", Formula["libbitcoin"].opt_libexec/"lib/pkgconfig"
-    ENV.prepend_path "PKG_CONFIG_PATH", Formula["libbitcoin-blockchain"].opt_libexec/"lib/pkgconfig"
-    ENV.prepend_create_path "PKG_CONFIG_PATH", libexec/"lib/pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", libexec/"lib/pkgconfig"
```

Comment:
This is a no-op if the path does not exist. So I think you've made it so that libbitcoin-node will not be able to find libbitcoin-network.

Reply:
Should I change it back to `prepend_create_path`? I find the documentation a little confusing on this.

Reply:
right